examples/game/breakout.rs

```diff
@@ -167,6 +175,10 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
     commands.spawn_bundle(OrthographicCameraBundle::new_2d());
     commands.spawn_bundle(UiCameraBundle::default());
 
+    // Sound
+    let brick_collision_sound = asset_server.load("sounds/breakout_collision.ogg");
```

Comment:
```suggestion
    let ball_collision_sound = asset_server.load("sounds/breakout_collision.ogg");
```

This sounds plays on collisions with the wall and paddle too.